e2e/cypress/tests/integration/enterprise/system_console/about/starter_edition_spec.js

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @enterprise @not_cloud @system_console @license_removal
+
+describe('System console', () => {
+    before(() => {
+        // * Ensure we are on self-hosted Starter edition
+        cy.shouldNotRunOnCloudEdition();
+        cy.apiDeleteLicense();
+    });
+
+    it('License page shows View plans button', () => {
```

Comment:
@yasserfaraazkhan we shall need to map this to zephyr

Reply:
Thank you Allan
I've added it to task: MM-45931

Reply:
Alright. Thanks a lot. Going to merge this.

Reply:
@AGMETEOR

added this test in zephyr. Will update the ID with one of the PR.

 [MM-T5132](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin:com.kanoah.test-manager__main-project-page#!/testCase/MM-T5132)